webpack/sass/main.scss

```diff
@@ -15,9 +18,21 @@ $brand-theme-light: $orange;
 $text-color: $jet-black;
 $project-card-theme: $blue;
 
-body {
-  font-family: 'Source Sans Pro', sans-serif;
-  color: $text-color;
+header {
+  @extend .padding-horizontal-xxl;
```

Comment:
as I said above, we should enclose the navbar inside a Bulma's `.container` element